docs/index.md

```diff
@@ -1,16 +1,31 @@
 Formula is a functional reactive framework built in Kotlin for managing state and side effects. It enables building 
 deterministic, composable, testable applications.
 
-### Quick example
-To demonstrate some of the concepts we will make a simple counter application. It shows the current count and 
+### Quick Start
+To demonstrate some of the concepts you will make a simple counter application. It shows the current count and 
 has two buttons to increment and decrement it.
 
 One of the best practises when working with Formula is to first think what the UI needs for rendering and what 
 actions the user will be able to take. This concept is called a `RenderModel` and is represented by a Kotlin data class.
 
+### Project Setup
+To get started create a new Android application called `Counter Application` with a blank activity. Next, rename (via refactor) 
+your `MainActivity` to `MyActivity`. Note: this is to make it easier for you to copy and past the example code later on.
+
+### Dependencies
+Now that you have your project set up, add the following libraries to your app level gradle file:
+
+```groovy
+dependencies {
+  implementation 'com.instacart.formula:formula-rxjava3:0.7.0'
+  implementation 'com.instacart.formula:formula-android:0.7.0'
+}
+```
+
 #### Render Model
 Render Model is an immutable representation of your view. It will be used to update Android views. Typically,
-it will also contain callbacks that will be invoked when user interacts with the UI.
+it will also contain callbacks that will be invoked when user interacts with the UI. Create a file called
```

Comment:
I wonder if we really need to be so explicit with `Create a file called
CounterRenderModel in the same package as your activity and add the following content.`